project_template/_base/project_name/apps/common/models/core.py

```diff
@@ -0,0 +1,54 @@
+from uuid import uuid4
+
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class CoreQuerySet(models.QuerySet):
+
+    def active(self):
+        return self.filter(is_active=True)
+
+    def inactive(self):
+        return self.filter(is_active=False)
+
+
+class CoreManager(models.Manager):
+
+    def get_queryset(self):
+        return CoreQuerySet(self.model, using=self._db)
+
+    def active(self):
+        return self.get_queryset().active()
+
+    def inactive(self):
+        return self.get_queryset().inactive()
+
+
+class CoreModel(models.Model):
```

Comment:
I our style guide I can't  find any recommendation about abstract model naming, but what do you think to add `Abstract` prefix here?

Reply:
Actually its @zada code from project developed two years age. I personally copy/pasted it in four or five projects and i thought from the beginning that it sounds really cool - `CoreModel`. Not some dull `BaseModel` or `CommonModel`.

I've got your point about `Abstract` prefix and it really make sense - we need to add this recommendation to the Code Style. But for this particular case I want to keep current name, either we can vote for it.

Reply:
`AbstractCoreModel` looks ugly and non-persistent with `CoreQuerySet` & `CoreManager` classes which could be used as is without inheritance. On one of the meetings we've agreed that use `CoreModel` without `Abstract` prefix.